install/threesdk/InstallTools.py

```diff
@@ -1873,19 +1873,41 @@ def _installbase_for_shell():
             Tools.execute(script, interactive=True, sudo_remove=sudoremove)
 
     @staticmethod
-    def get_newest_version():
-        from threesdk import __version__
+    def is_latest_release():
+        try:
+            from threesdk import __version__
+        except:
+            from .core import __version__
+        # call releases api
+        resp = requests.get("https://api.github.com/repos/threefoldtech/jumpscaleX_core/releases/latest")
+        resp = resp.json()
+        latest_release = resp["tag_name"]
+        if latest_release is not __version__:
+            return False
+        return True
 
+    @staticmethod
+    def get_latest_release():
+        try:
```

Comment:
do you need __version here?

Reply:
no don't need yet, deleted unused variable :+1: